apps/templates/content_templates.py

```diff
@@ -197,13 +197,16 @@ def get(self, req, lookup):
         privileges = active_user.get('active_privileges', {})
         if not lookup:
             lookup = {}
-        if not privileges.get('content_templates') and not privileges.get('personal_template'):
+        manage = req.args.get('manage')
+
+        if manage and not privileges.get('content_templates') and not privileges.get('personal_template'):
             lookup.update({'$and': [{'is_public': False}, {'user': active_user.get('_id')}]})
-        elif not privileges.get('content_templates') and privileges.get('personal_template'):
+        elif manage and not privileges.get('content_templates') and privileges.get('personal_template'):
             lookup.update({'$or': [{'is_public': False}, {'user': active_user.get('_id')}]})
-        elif privileges.get('content_templates') and not privileges.get('personal_template'):
+        elif (manage and privileges.get('content_templates')
+                and not privileges.get('personal_template')
+                or not manage and not privileges.get('personal_template')):
```

Comment:
this condition seems kind of complicated, can you pls split the manage and not manage part